custom-words.txt

```diff
@@ -16,6 +16,7 @@ ACCELCONTAINER
 acceptors
 accesskey
 accesspoint
+accesspolicies
```

Comment:
Adding this term shouldn't needed, you should be able to replace it with 'accessPolicies' 'AccessPolicies' or 'access policies' everywhere.